src/ismn/custom.py

```diff
@@ -82,21 +82,21 @@ class CustomStationMetadataCsv(CustomMetaReader):
 
     def __init__(self, station_meta_csv, fill_values=None, **kwargs):
         """
-        Parameters
-        ----------
-        station_meta_csv: str
-            Path to the csv file with the above described content
-       fill_values: dict, optional (default: None)
-            Values to use for a certain custom metadata variable, if no
-            match is found.
-        kwargs:
-            Additional kwargs as passed to :func:`pandas.read_csv`
-            To use a different separator than the default semicolon, use `sep`
+         Parameters
```

Comment:
This looks not intentional

Reply:
I don't think I did that. Just checked the TUW-GEO/ismn repo and it already contains this piece of code, last committed 4 months ago. If I did it, then completely unintentionally

Reply:
ah true